### ISSUE TITLE
Add requirements.txt

### ISSUE THREAD

**Issue by dhruvaop** [Bauddhik-Geeks/ATOM-The-Voice-Assistant#4]
You can also add requirements.txt of all the libraries used in python

**Comment by dhruvaop** [comment]
absl-py==0.13.0
appdirs==1.4.4
astunparse==1.6.3
attrs==21.2.0
audioread==2.1.9
autopep8==1.5.7
backports.entry-points-selectable==1.1.0
beautifulsoup4==4.9.3
bs4==0.0.1
cachetools==4.2.2
certifi==2021.5.30
cffi==1.14.6
charset-normalizer==2.0.4
clang==5.0
click==8.0.1
colorama==0.4.4
comtypes==1.1.10
cycler==0.10.0
decorator==5.1.0
dill==0.3.4
distlib==0.3.2
docopt==0.6.2
docutils==0.17.1
ffmpeg==1.4
filelock==3.0.12
flatbuffers==1.12
gast==0.4.0
google-auth==1.35.0
google-auth-oauthlib==0.4.5
google-pasta==0.2.0
grpcio==1.39.0
gTTS==2.2.3
h5py==3.1.0
idna==3.2
imutils==0.5.4
inflect==5.3.0
joblib==1.0.1
Js2Py==0.71
jsonpatch==1.32
jsonpointer==2.1
keras==2.6.0
Keras-Preprocessing==1.1.2
Kivy==2.0.0
kivy-deps.angle==0.3.0
kivy-deps.glew==0.3.0
kivy-deps.sdl2==0.3.1
Kivy-examples==2.0.0
Kivy-Garden==0.1.4
kiwisolver==1.3.1
librosa==0.8.1
llvmlite==0.37.0
Markdown==3.3.4
matplotlib==3.4.3
mediapipe==0.8.7.1
MouseInfo==0.1.3
multiprocess==0.70.12.2
numba==0.54.0
numpy==1.19.3
oauthlib==3.1.1
opencv-contrib-python==4.5.3.56
opencv-python==4.5.3.56
opt-einsum==3.3.0
packaging==21.0
pandas==1.3.2
Pillow==8.3.1
pipwin==0.5.1
platformdirs==2.3.0
playsound==1.3.0
playsound2==0.1
pooch==1.5.1
protobuf==3.17.3
pyasn1==0.4.8
pyasn1-modules==0.2.8
PyAudio 
PyAutoGUI==0.9.53
pycodestyle==2.7.0
pycparser==2.20
PyGetWindow==0.0.9
Pygments==2.10.0
pyjsparser==2.7.1
PyMsgBox==1.0.9
pynndescent==0.5.4
pyparsing==2.4.7
pyperclip==1.8.2
pypiwin32==223
PyPrind==2.11.3
PyQt5==5.15.4
PyQt5-Qt5==5.15.2
PyQt5-sip==12.9.0
PyRect==0.1.4
PyScreeze==0.1.27
pySmartDL==1.3.4
python-dateutil==2.8.2
pyttsx3==2.90
PyTweening==1.0.3
pytz==2021.1
pywin32==301
pyzmq==22.2.1
requests==2.26.0
requests-oauthlib==1.3.0
resampy==0.2.2
rsa==4.7.2
scikit-learn==0.24.2
scipy==1.7.1
six==1.15.0
sklearn==0.0
sounddevice==0.4.2
SoundFile==0.10.3.post1
soupsieve==2.2.1
SpeechRecognition==3.8.1
tensorboard==2.6.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
tensorflow==2.6.0
tensorflow-estimator==2.6.0
termcolor==1.1.0
threadpoolctl==2.2.0
toml==0.10.2
torch==1.9.0
torchfile==0.1.0
torchvision==0.10.0
tornado==6.1
tqdm==4.62.2
typing-extensions==3.7.4.3
tzdata==2021.1
tzlocal==3.0
umap-learn==0.5.1
Unidecode==1.3.1
urllib3==1.26.6
virtualenv==20.7.2
visdom==0.1.8.9
websocket-client==1.2.1
Werkzeug==2.0.1
wrapt==1.12.1

**Comment by dhruvaop** [comment]
pip install -r requirements.txt

**Comment by Nitesh-thapliyal** [comment]
> absl-py==0.13.0
> appdirs==1.4.4
> astunparse==1.6.3
> attrs==21.2.0
> audioread==2.1.9
> autopep8==1.5.7
> backports.entry-points-selectable==1.1.0
> beautifulsoup4==4.9.3
> bs4==0.0.1
> cachetools==4.2.2
> certifi==2021.5.30
> cffi==1.14.6
> charset-normalizer==2.0.4
> clang==5.0
> click==8.0.1
> colorama==0.4.4
> comtypes==1.1.10
> cycler==0.10.0
> decorator==5.1.0
> dill==0.3.4
> distlib==0.3.2
> docopt==0.6.2
> docutils==0.17.1
> ffmpeg==1.4
> filelock==3.0.12
> flatbuffers==1.12
> gast==0.4.0
> google-auth==1.35.0
> google-auth-oauthlib==0.4.5
> google-pasta==0.2.0
> grpcio==1.39.0
> gTTS==2.2.3
> h5py==3.1.0
> idna==3.2
> imutils==0.5.4
> inflect==5.3.0
> joblib==1.0.1
> Js2Py==0.71
> jsonpatch==1.32
> jsonpointer==2.1
> keras==2.6.0
> Keras-Preprocessing==1.1.2
> Kivy==2.0.0
> kivy-deps.angle==0.3.0
> kivy-deps.glew==0.3.0
> kivy-deps.sdl2==0.3.1
> Kivy-examples==2.0.0
> Kivy-Garden==0.1.4
> kiwisolver==1.3.1
> librosa==0.8.1
> llvmlite==0.37.0
> Markdown==3.3.4
> matplotlib==3.4.3
> mediapipe==0.8.7.1
> MouseInfo==0.1.3
> multiprocess==0.70.12.2
> numba==0.54.0
> numpy==1.19.3
> oauthlib==3.1.1
> opencv-contrib-python==4.5.3.56
> opencv-python==4.5.3.56
> opt-einsum==3.3.0
> packaging==21.0
> pandas==1.3.2
> Pillow==8.3.1
> pipwin==0.5.1
> platformdirs==2.3.0
> playsound==1.3.0
> playsound2==0.1
> pooch==1.5.1
> protobuf==3.17.3
> pyasn1==0.4.8
> pyasn1-modules==0.2.8
> PyAudio
> PyAutoGUI==0.9.53
> pycodestyle==2.7.0
> pycparser==2.20
> PyGetWindow==0.0.9
> Pygments==2.10.0
> pyjsparser==2.7.1
> PyMsgBox==1.0.9
> pynndescent==0.5.4
> pyparsing==2.4.7
> pyperclip==1.8.2
> pypiwin32==223
> PyPrind==2.11.3
> PyQt5==5.15.4
> PyQt5-Qt5==5.15.2
> PyQt5-sip==12.9.0
> PyRect==0.1.4
> PyScreeze==0.1.27
> pySmartDL==1.3.4
> python-dateutil==2.8.2
> pyttsx3==2.90
> PyTweening==1.0.3
> pytz==2021.1
> pywin32==301
> pyzmq==22.2.1
> requests==2.26.0
> requests-oauthlib==1.3.0
> resampy==0.2.2
> rsa==4.7.2
> scikit-learn==0.24.2
> scipy==1.7.1
> six==1.15.0
> sklearn==0.0
> sounddevice==0.4.2
> SoundFile==0.10.3.post1
> soupsieve==2.2.1
> SpeechRecognition==3.8.1
> tensorboard==2.6.0
> tensorboard-data-server==0.6.1
> tensorboard-plugin-wit==1.8.0
> tensorflow==2.6.0
> tensorflow-estimator==2.6.0
> termcolor==1.1.0
> threadpoolctl==2.2.0
> toml==0.10.2
> torch==1.9.0
> torchfile==0.1.0
> torchvision==0.10.0
> tornado==6.1
> tqdm==4.62.2
> typing-extensions==3.7.4.3
> tzdata==2021.1
> tzlocal==3.0
> umap-learn==0.5.1
> Unidecode==1.3.1
> urllib3==1.26.6
> virtualenv==20.7.2
> visdom==0.1.8.9
> websocket-client==1.2.1
> Werkzeug==2.0.1
> wrapt==1.12.1

**Comment by Nitesh-thapliyal** [comment]
@dhruvaop In requirement.txt add the libraries, and software that are required to run the code in local system.

**Comment by DhruvaBhattacharya** [comment]
@Nitesh-thapliyal you can use pip freeze requirements.txt to get the file

**Comment by dhruvaop** [comment]
@Nitesh-thapliyal you can use pip freeze requirements.txt to get the file
